apps/backend/src/modules/auth/auth.controller.ts

```diff
@@ -63,10 +77,25 @@ class AuthController extends BaseController {
 	 *              schema:
 	 *                type: object
 	 *                properties:
-	 *                  message:
-	 *                    type: object
+	 *                  user:
 	 *                    $ref: "#/components/schemas/User"
+	 *                  token:
+	 *                    type: string
+	 *                    description: "Authentication token for the user."
+	 *                    example: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ..."
 	 */
+
+	private async signIn(
+		options: APIHandlerOptions<{
+			body: UserSignUpRequestDto;
```

Comment:
```suggestion
			body: UserSignInRequestDto;
```

This is a signIn route